### PR TITLE
Fixed #33789 -- Doc'd changes in quoting table/column names on Oracle in Django 4.0.

### PR DESCRIPTION
**Pull request by felixxm**
Thanks Paul in 't Hout for the report.

Regression in 1f643c28b5f2b039c47155692844dbae1cb091cd.

ticket-33789

### CODE REVIEW

File: docs/releases/4.0.txt [hunk 1]
```diff
@@ -492,6 +492,15 @@ some cases.
 consequence, any custom deletion logic in ``delete()`` handlers should be
 moved to ``form_valid()``, or a shared helper method, if required.
 
+Table and column naming scheme changes on Oracle
+------------------------------------------------
+
+Django 4.0 inadvertently changed the table and column naming scheme on Oracle.
+This causes errors for models and fields with names longer than 30 characters.
+Unfortunately, it necessarily requires renaming some Oracle tables and columns.
```
Comment (smithdc1) [inline docs/releases/4.0.txt:500]:
```suggestion
Unfortunately, renaming some Oracle tables and columns is required.
```

Comment (felixxm) [reply docs/releases/4.0.txt]: Thanks, updated :+1:

**Review by carltongibson — APPROVED** [review]
Great thanks.

**Comment by felixxm** [comment]
Thanks both :star:

**Comment by rptmat57** [comment]
I apologize for digging this old thread but I am a bit confused. If I update from <4.0 to >4.0, for example 3.2 to 4.2, do I need to run the script?

**Pull request merged**